### ISSUE TITLE
[3.1.1] Bitbucket support seems currently broken

### ISSUE THREAD

**Issue by risinga** [eddiewebb/circleci-queue#121]
### Orb version

3.1.1

### What happened

When using bitbucket, the orb isn't working. The regular expression is not identifying in the bash script something like this:
git@bitbucket.org:deedee/deeedee.git as a bitbucket address

Current code:
```
 VCS_TYPE="github"
    if [[ "$CIRCLE_REPOSITORY_URL" =~ "*bitbucket.org*" ]]; then
        VCS_TYPE="bitbucket"
    fi
```
But instead, it's assuming the default github VCS so consequently failing to reach to the project URL correctly

Should change to something like the following, so it could accept any: 

### Expected behavior

```

VCS_TYPE="github"
if [[ "$CIRCLE_REPOSITORY_URL" =~ .*bitbucket\.org.* ]]; then
    VCS_TYPE="bitbucket"
fi
```

**Comment by risinga** [comment]
I'm happy to submit the fix if folks are ok with it

**Comment by eddiewebb** [comment]
Happy to receive a PR, thank you!

**Comment by risinga** [comment]
Awesome. I just got a permissions issue trying to push the branch. Could you add me to the contributors list please?

**Comment by eddiewebb** [comment]
@risinga - fork the repo and open PR from your own copy.

- https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/working-with-forks/fork-a-repo
- https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/proposing-changes-to-your-work-with-pull-requests/creating-a-pull-request-from-a-fork

**Comment by risinga** [comment]
Wonderful! Thanks for the links. Needed a refresher on the github ways :)

**Comment by eddiewebb** [comment]
#122 is merged and published as `3.1.2` currently. Thank you @risinga !
